sensing/pointcloud_preprocessor/src/blockage_diag/blockage_diag_nodelet.cpp

```diff
@@ -44,8 +44,15 @@ BlockageDiagComponent::BlockageDiagComponent(const rclcpp::NodeOptions & options
     dust_kernel_size_ = declare_parameter<int>("dust_kernel_size");
     dust_buffering_frames_ = declare_parameter<int>("dust_buffering_frames");
     dust_buffering_interval_ = declare_parameter<int>("dust_buffering_interval");
+    max_distance_range_ = declare_parameter<double>("max_distance_range");
+    horizontal_resolution_ = declare_parameter<double>("horizontal_resolution");
```

Comment:
[Q]
Does horizontal_resolution mean azimuth resolution? Should this parameter be adjusted to align with the sensor?
Could you write the meaning of this parameter in document?

Reply:
`horizontal_resolution` is related to resolutoin of depth map image. I updated the document at https://github.com/autowarefoundation/autoware.universe/pull/5932/commits/fea22ccb1e412d96cc526c36aebca661403921a0